packages/webview_flutter/webview_flutter_platform_interface/lib/src/types/http_response_error.dart

```diff
@@ -38,8 +41,16 @@ class HttpResponseError {
   /// Used by the platform implementation to create a new [HttpResponseError].
   const HttpResponseError({
     required this.statusCode,
+    this.request,
+    this.response,
   });
 
   /// The HTTP status code.
   final int statusCode;
```

Comment:
I think this field should be moved into `WebResourceResponse`. This is a breaking change, but this class is unused by the implementations yet. So I don't believe anyone is using this class yet. 

@stuartmorgan FYI if you disagree